packages/react-charting/src/components/PieChart/Arc/Arc.types.ts

```diff
@@ -16,11 +16,11 @@ export interface IArcProps {
   /**
    * innerRadius of the Arc.
    */
-  innerRadius?: number;
+  innerRadius: number;
```

Comment:
Can this change break existing users who have not defined inner/outer radius.

Reply:
Inner Radius is a hardcoded value, and Arc is an internal component

Reply:
it has a hardcoded value of 1, currently, earlier it was 0.